services/src/atdd/src/test/java/org/openkilda/atdd/FlowPathTest.java

```diff
@@ -147,18 +147,19 @@ public void checkAlternativePathAvailableBandwidthDecreased(int expectedAvailabl
     public void flowPathCorrect(String flowId, String sourceSwitch, int sourcePort, int sourceVlan,
             String destinationSwitch, int destinationPort, int destinationVlan, int bandwidth)
             throws UnroutablePathException, InterruptedException, RecoverableException {
-        Flow flow = new Flow(FlowUtils.getFlowName(flowId), bandwidth, false, flowId, sourceSwitch,
-                sourcePort, sourceVlan, destinationSwitch, destinationPort, destinationVlan);
+        Flow flow =
+                new Flow(FlowUtils.getFlowName(flowId), bandwidth, false, flowId, new SwitchId(sourceSwitch),
+                sourcePort, sourceVlan, new SwitchId(destinationSwitch), destinationPort, destinationVlan);
         ImmutablePair<PathInfoData, PathInfoData> path = FlowUtils.getFlowPath(flow);
         System.out.println(path);
         assertEquals(expectedShortestPath, path);
     }
 
-    private int getBandwidth(int expectedBandwidth, String src_switch, String src_port) throws InterruptedException {
-        int actualBandwidth = getLinkBandwidth(src_switch, src_port);
+    private int getBandwidth(int expectedBandwidth, String srcSwitch, String srcPort) throws InterruptedException {
```

Comment:
Why `srcSwitch` is a `String`? If it become StringId you can remove .toString() from caller sides... But you will need to convert rest of predefined constants into StringId too.

Reply:
You are right. I will fix it.